extension.json

```diff
@@ -62,19 +62,25 @@
 		"Miraheze\\CreateWiki\\Maintenance\\RenameWiki": "maintenance/renameWiki.php"
 	},
 	"TestAutoloadNamespaces": {
+		"Miraheze\\CreateWiki\\Tests\\Unit\\": "tests/phpunit/unit",
 		"Miraheze\\CreateWiki\\Tests\\": "tests/phpunit/"
 	},
 	"Hooks": {
 		"BeforeCreateEchoEvent": [
```

Comment:
This can be a single line (not array). Optional to do though.

Reply:
Actually can't this use the HookHandler?

Reply:
> Actually can't this use the HookHandler?

I believe Echo does not yet support that way yet.

Reply:
Any hook should so long as it uses `public function on{hookName}` it doesn't need an interface to support it as fat as I'm aware though could be wrong.

Reply:
> This can be a single line (not array). Optional to do though.

Done!

Reply:
> Any hook should so long as it uses `public function on{hookName}` it doesn't need an interface to support it as far as I'm aware though could be wrong.

I'm not sure what I should do if accept your request. If you OK, can I do that in another PR?

Reply:
Another PR is fine, yeah.